### ISSUE TITLE
[IMPROVEMENT] - FriendNotifications - 

### ISSUE THREAD

**Issue by Dragonisser** [mwittrien/BetterDiscordAddons#1500]
**Describe the solution you'd like**
Currently the Login/-out Timelog isnt saved permanently and gets deleted when restarting discord. 
It would be nice if it was saved as a log/json file in the plugins folder so it can be loaded on restart of discord or to simply have it as a log.

**Describe alternatives you've considered**
None, would require to edit the plugin myself, which i could, but i rather want to have it official.

**Additional context**
- Toggle under the settings to enable the permanent log
- Delete entries after x days or not at all

**Comment by mwittrien** [comment]
No, the plugin isnt meant to document the login behaviour of your friends
